app/controllers/datasets/docs_controller.rb

```diff
@@ -12,6 +13,7 @@ def new
 
   def create
     @doc = @dataset.docs.build(doc_params)
+    @doc.documentation = true
```

Comment:
I get what you are trying to do here but I wonder if having an attribute of 'type' could be worth considering. That way, it's extendable ...

e.g type='document'

Single Table inheritance would automate this also ...

Reply:
Yes, this is a temporary measure. 'documentation' isn't great, but it's already in the datafile model, apparently unused. Agree with you on improving, but probably later